packages/shared/ReactSymbols.js

```diff
@@ -24,8 +24,8 @@ export const REACT_STRICT_MODE_TYPE = hasSymbol
   ? Symbol.for('react.strict_mode')
   : 0xeacc;
 export const REACT_PROFILER_TYPE = hasSymbol
-  ? Symbol.for('react.profile_root')
-  : 0xeacc;
```

Comment:
Wonder if there's some sort of automated test we could write to guard against this sort of clowniness in the future.

Reply:
Object.values(require('ReactSymbols')) in a jest test maybe?